### PR TITLE
fix(side-panel): Fix keyframes animation

### LINKED ISSUE #934: SidePanel (LABS) expanding animation triggered (unwanted)
## 🐛 Bug Report

Canvas-kit Labs Side Panel component is triggering an animation (expanding) even if panel is already expanded (when clicking for first time in menu and modifying state). See video below and also example referenced via codesandbox.io

## To Reproduce
One of many ways how to reproduce it when using the component in your code.
1. Render SidePanel component
2. Render children inside => i.e \<divs>
3. Attach onClick handler to \<divs> and modify state inside handler (for example selected div index)
4. Observe when this component renders and you click for first time on the \<div>, Panel triggers animation

## Expected Behavior

Do not trigger animation for already expanded Panel.

## Actual Results

Animation is triggered.

## Browser (if applicable)

Latest Chrome (v87.x)

## Link to repl or repo (highly encouraged)

codesandbox example:
https://codesandbox.io/s/canvas-kit-sidepanel-animation-issue-5p75o?file=/src/App.tsx

### Error Output

https://user-images.githubusercontent.com/7189601/103699531-605e7a00-4fa3-11eb-9e9f-aa5da04719f2.mov

### PR DESCRIPTION
**Pull request by alanbsmith**
## Summary

Resolves #934 

## Checklist

- [ ] branch has been rebased on the latest master commit
- [ ] tests are changed or added
- [ ] `yarn test` passes
- [ ] all (dev)dependencies that the module needs is added to its `package.json`
- [ ] code has been documented and, if applicable, usage described in README.md
- [ ] module has been added to `canvas-kit-react` and/or `canvas-kit-css` universal modules, if
      applicable
- [ ] design approved final implementation
- [ ] a11y approved final implementation
- [ ] code adheres to the [API & Pattern guidelines](../API_PATTERN_GUIDELINES.md)

### CODE REVIEW

File: modules/_labs/side-panel/react/lib/SidePanel.tsx [hunk 1]
```diff
@@ -59,6 +59,7 @@ export interface SidePanelProps extends React.HTMLAttributes<HTMLDivElement> {
    * @default 'standard'
    */
   variant?: SidePanelVariant;
+  touched?: boolean;
```
Comment (alanbsmith) [inline modules/_labs/side-panel/react/lib/SidePanel.tsx:62]: TODO: Add JSDoc info here

File: modules/_labs/side-panel/react/lib/SidePanel.tsx [hunk 2]
```diff
@@ -114,19 +115,13 @@ const SidePanel = ({
   onStateTransition,
   origin = 'left',
   variant = 'standard',
+  touched,
```
Comment (alanbsmith) [inline modules/_labs/side-panel/react/lib/SidePanel.tsx:118]: I'm open to other names for this prop. It really means "the expanded state of the component has had intentional interaction.

Comment (NicholasBoll) [reply modules/_labs/side-panel/react/lib/SidePanel.tsx:118]: I'm okay with this. Add some JSDoc and I think that's good. I'm not sure there's a more succinct word to describe this variable.

**Review by NicholasBoll — APPROVED** [review]

File: modules/_labs/side-panel/react/lib/SidePanel.tsx (hunk 2 above)
Comment (alanbsmith) [reply modules/_labs/side-panel/react/lib/SidePanel.tsx:123]: Done. Thanks!

**Pull request merged**